Proxy/Configuration/Response/Json/index.ts

```diff
@@ -110,6 +112,17 @@ export namespace Json {
 			expires: month && year ? [month, year] : [0, 0],
 		}
 	}
+
+	export function extractToken(configuration: Json, body: Record<string, any>): string | undefined {
+		let result: string | undefined
+		if (!configuration.token) {
+			result = undefined
+		} else {
+			result = Selector.get<string>(body, configuration.token)
+		}
```

Comment:
It should be possible to just do a general regexp search on the whole body for tokens and replace them, or?

Reply:
Yes, might be easier.